src/Private/Get-WindowsAppId.ps1

```diff
@@ -0,0 +1,12 @@
+# Helper function to get the relevant AppId for the calling PowerShell edition
+
+function Get-WindowsAppId {
+    if ($PSEdition -eq 'Desktop'){
+        (Get-StartApps -Name 'PowerShell').where({$_.Name -eq 'Windows PowerShell'}).AppId
+    } elseif ($IsWindows -and $PSVersionTable.PSVersion -ge [Version]'6.1') {
```

Comment:
This code path will only get called on Windows so the `IsWindows` isn't needed but it also doesn't hurt too much...

I don't care either way but I'm leaning on taking it out.

Reply:
I think at one point I was thinking of a fall back if it was PS Core, but not a new enough version to use `-SkipEditionCheck`...

Checking the version number is enough to get the job done here. Will remove the IsWindows Part.

Reply:
That's a good point about `-SkipEditionCheck`. I think this is fine the way it is. As time goes on, we wont have to worry about PowerShell Core 6.0